ui/client/common/JsonUtils.js

```diff
@@ -16,6 +16,14 @@ export function tryParse(input){
   }
 }
 
+export function tryParseOrNull(input) {
```

Comment:
Mayby better will be parseOrNull?

Reply:
It is consistent with the above function `tryParse`.

Reply:
But in your function you parse or return null - not trying :)